tools/codegen/schema/testdata/primitive-attributes.golden.go

```diff
@@ -1,4 +1,4 @@
-package test_name
+package testname
```

Comment:
it's idiomatic Go to call the package same as last folder, here it is testdata but testname

Reply:
This actually a golden file to verify the output of generated code. In the test the resource name is defined as `test_name` so this package is now accurate.